buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -65,6 +66,8 @@ class FladlePluginDelegate {
   private fun TaskContainer.createTasksForConfig(base: FlankGradleExtension, config: FladleConfig, project: Project, name: String) {
     checkIfSanityAndValidateConfigs(config)
     validateOptionsUsed(config = config, flank = base.flankVersion.get())
+    val resultsDir = if (config.localResultsDir.isPresent) null
```

Comment:
looks pretty good. can we wrap this if statement inside a function?

Reply:
actually reading this a second time, should it be like this?
```suggestion
    val resultsDir = if (config.localResultsDir.isPresent) config.localResultsDir.get()
```